demo/store/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@store",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
Could we please describe the purpose of this store package here?

Reply:
And we should add `"private": true` to store, modules and screen's package.json files.

Reply:
Done in d1adee8 and made the other stuff private in  e3db2f1